google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -759,6 +772,19 @@ public boolean isRefreshingChannel() {
       return primedTableIds;
     }
 
+    /** Configure the header tracer for surfacing metrics in the header. */
```

Comment:
We should avoid exposing this to customers. It's an implementation detail for the current approach. When this gets pushed into gax, we will move this into ApiTracer so we should minimize the amount public methods exposed (even if Beta). if this needs to be public then it should be INternalApi

Reply:
Updated functions to protected.